### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#123]
Zeus version:
`1.0.29.1f59`

Error info:
```Traceback (most recent call last):
  File "zeus.py", line 427, in <module>
    auto=opt.autoStartSqlmap, verbose=opt.runInVerbose, batch=opt.runInBatch
  File "zeus.py", line 378, in __run_attacks
    return sqlmap_scan.sqlmap_scan_main(url.strip(), verbose=verbose, opts=__create_arguments(sqlmap=True),
  File "zeus.py", line 327, in __create_arguments
    option = (to_use[0], to_use[1])
IndexError: list index out of range
````

Running details:
`Linux-4.12.0-kali2-amd64-x86_64-with-Kali-kali-rolling-kali-rolling`

Commands used:
`zeus.py -d inurl:account.asp?p= -s -x --sqlmap-args=getUsers, level 5, risk 3, verbose, randomAgent, batch, keepAlive`

Log file info:
```2017-10-22 18:12:48,526;zeus-log;INFO;log file being saved to '/root/Zeus-Scanner/log/zeus-log-16.log'...
2017-10-22 18:12:48,527;zeus-log;INFO;using default search engine...
2017-10-22 18:12:48,527;zeus-log;INFO;starting dork scan with query 'inurl:account.asp?p='...
2017-10-22 18:12:48,527;zeus-log;INFO;attempting to gather query URL...
2017-10-22 18:12:48,761;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-22 18:12:48,767;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-22 18:12:55,820;zeus-log;INFO;browser will open shortly...
2017-10-22 18:12:58,059;zeus-log;INFO;searching 'http://google.com' using query 'inurl:account.asp?p='...
2017-10-22 18:13:01,430;zeus-log;INFO;closing the browser and continuing process..
2017-10-22 18:13:01,791;zeus-log;INFO;URL successfully gathered, searching for GET parameters...
2017-10-22 18:13:01,791;zeus-log;INFO;no proxy configuration detected...
2017-10-22 18:13:02,520;zeus-log;INFO;adjusting user-agent header to default user agent 'Zeus-Scanner(v1.0.29.1f59)::Python->v2.7'...
2017-10-22 18:13:02,523;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,524;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,524;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,524;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,525;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,525;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,525;zeus-log;INFO;received webcache URL, extracting URL from webcache...
2017-10-22 18:13:02,525;zeus-log;INFO;found a total of 18 URL's with a GET parameter...
2017-10-22 18:13:02,526;zeus-log;INFO;successfully wrote found items to '/root/Zeus-Scanner/log/url-log/url-log-2.log'...
2017-10-22 18:13:07,442;zeus-log;WARNING;skipping 'http://www.investopedia.com/terms/a/account.asp%2Binurl'...
2017-10-22 18:13:08,922;zeus-log;WARNING;skipping 'http://searchcode.com/file/44385705/user/goahead/web/services/account.asp%2Binurl'...
2017-10-22 18:13:09,747;zeus-log;WARNING;skipping 'https://webstore.ridecj.com/ticketing/create-account.asp?p=/ticketing/Default.asp&wg=&curr=I5qLYkyLZU-LaFKha1WUlW9ZpplzXZyrqHhir7CtfWemtoFrp7qFb7yvv4p0wbHEj3m1vsi7lX-7xM6-m4XBytTVyKKMyNHb3Mypk92sltmv'...
2017-10-22 18:13:10,467;zeus-log;WARNING;skipping 'https://access.lockheedmartin.com/account.asp+inurl:account.asp?p='...
2017-10-22 18:13:11,250;zeus-log;WARNING;skipping 'http://maps.google.nl/maps?hl=nl&tab=wl'...
2017-10-22 18:13:11,882;zeus-log;WARNING;skipping 'http://www.google.nl/search?hl=nl&tbm=isch&source=og&tab=wi'...
2017-10-22 18:13:12,506;zeus-log;WARNING;skipping 'http://www.investopedia.com/terms/t/t-account.asp%2Binurl'...
2017-10-22 18:13:13,242;zeus-log;WARNING;skipping 'http://maps.google.nl/maps?hl=nl'...
2017-10-22 18:13:13,835;zeus-log;WARNING;skipping 'http://news.google.nl/nwshp?hl=nl&tab=wn'...
2017-10-22 18:13:14,755;zeus-log;WARNING;skipping 'https://webstore.concordcoachlines.com/ticketing/create-account.asp?p=/ticketing/view-map.asp&curr=beTVrJbVr5nVspzrtZ_e37mj8OO9p-b18sKs-fr3x7HwAMu18QTPuQb5CdS-C_sO2cP_CBIF38kFDhgI5c8LFB4fEuzWEhslJhbz3Sf24CP5&wg='...
2017-10-22 18:13:58,475;zeus-log;INFO;creating arguments for sqlmap...
2017-10-22 18:13:58,475;zeus-log;ERROR;[7;31;31mran into exception 'list index out of range' exception has been saved to log file...
Traceback (most recent call last):
  File "zeus.py", line 427, in <module>
    auto=opt.autoStartSqlmap, verbose=opt.runInVerbose, batch=opt.runInBatch
  File "zeus.py", line 378, in __run_attacks
    return sqlmap_scan.sqlmap_scan_main(url.strip(), verbose=verbose, opts=__create_arguments(sqlmap=True),
  File "zeus.py", line 327, in __create_arguments
    option = (to_use[0], to_use[1])
IndexError: list index out of range

2017-10-22 18:14:25,163;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-22 18:14:25,164;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-22 18:14:25,184;zeus-log;INFO;[32mextracting traceback from log file...[0m
```

**Comment by Ekultek** [comment]
Firstly, you need to update, run `python zeus.py --update`.

Secondly, the commands need to be passed with `true` or `false` wrapped in quotes, for example with yours:

`zeus.py -d inurl:account.asp?p= -s -x --sqlmap-args="getUsers true, level 5, risk 3, verbose 2, randomAgent true, batch true, keepAlive true"`

I have an idea on how I can fix this issue though so give me a few minutes please.

**Comment by Ekultek** [comment]
Patched via d590ca2 it will not automatically include `true` if the argument is passed without an external argument.
